modules/SofaMiscMapping/CMakeLists.txt

```diff
@@ -71,7 +71,12 @@ set(SOURCE_FILES
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 target_link_libraries(${PROJECT_NAME} PUBLIC SofaSimpleFem SofaRigid)
+
+if(USE_COMPRESSED_ROW_SPARSE)
```

Comment:
My bad...  It was an intermediate code that was useful to get getJs but we no longer use this function
This will be removed from the pull request